Classes/Util/PBOpenFiles.h

```diff
@@ -8,9 +8,11 @@
 
 #import <Foundation/Foundation.h>
 
+@class PBChangedFile;
+
 @interface PBOpenFiles : NSObject
 
-+ (void)showInFinderAction:(id)sender with:(NSURL *)workingDirectoryURL;
-+ (void)openFilesAction:(id)sender with:(NSURL *)workingDirectoryURL;
++ (void)openFiles:(NSArray<PBChangedFile *> *)selectedFiles with:(NSURL *)workingDirectoryURL;
```

Comment:
That code is supposed to work with `NSString` or `PBChangedFile` and whatever you can throw at it (hence -selectedURLsFromSender:...`, so I'd prefer to keep it that way :wink: